be/src/runtime/load_channel_mgr.h

```diff
@@ -80,7 +89,8 @@ class LoadChannelMgr {
     Cache* _last_success_channel = nullptr;
 
     // check the total load channel mem consumption of this Backend
-    std::shared_ptr<MemTrackerLimiter> _mem_tracker;
+    std::unique_ptr<MemTracker> _mem_tracker;
```

Comment:
I think we should use MemTrackerLimiter here. Normal memtracker could be used to control flush memtable. But if it could not flush memtable accurately then limiter memtracker will work. If this is a normal memtracker, OOM will occurred.

Reply:
Ditto~